roles/thanos_sidecar/meta/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+galaxy_info:
+  author: Matej Feder
+  description: Role osism.services.thanos_sidecar
+  company: OSISM GmbH
```

Comment:
Feel free to use dNation here.

Reply:
thx, see b8f8cab86508dd2ed0a80416a4cf72a5a1fce3f2